src/LibHdb++.cpp

```diff
@@ -1,40 +1,44 @@
-//=============================================================================
-//
-// file :        LibHdbL.cpp
-//
-// description : Source for the LibHdb library.
-//
-// Author: Graziano Scalamera
-//
-//
-//=============================================================================
+/* Copright (C) 2014-2017
+   Elettra - Sincrotrone Trieste S.C.p.A.
+   Strada Statale 14 - km 163,5 in AREA Science Park
+   34149 Basovizza, Trieste, Italy.
 
-#include "LibHdb++.h"
-#include <dlfcn.h>
+   This file is part of libhdb++.
 
-#ifndef LIB_BUILDTIME
-#define LIB_BUILDTIME   RELEASE " " __DATE__ " "  __TIME__
-#endif
+   libhdb++cassandra is free software: you can redistribute it and/or modify
```

Comment:
Here it is libhdb++, not libhdb++cassandra (please check on all the files and all the projects) ;-)
There is the same mistake a few lines below too.

Reply:
Ouch, copy and paste error!